### DISCUSSION TITLE
편집하기, 완료하기, 추가하기 페이지 통합

### DISCUSSION THREAD

**Discussion by codeisneverodd** [prgrms-fe-devcourse/FEDC2_TimeDivider_Root#32] [0. 개발]
피그마에 페이지 흐름상 다른 페이지처럼 표현이 되어있지만, 실제로는 모달이 있다 없다 하는 하나의 페이지 입니다.
모달 관련 작업을 하던중 모달을 껐을 때 페이지가 이동되도록 구현하는 것이 어색하다고 생각되어 3개의 페이지를 하나로 통합하고자 합니다! 의견 남겨주세요!!
@ljw0096 @dahye1013

**Comment by dahye1013** [comment]
좋아요~ 
원래 초반에 상의할때도 수정하기 화면으로 가게 라우팅 처리하는걸로 하기로 했는데,
피그마 이미지만보고 페이지를 만들어버렸네요ㅎㅎㅎ..!
수정해주신것 확인했습니다 !
